.drone.yml

```diff
@@ -68,8 +68,8 @@ pipeline:
     environment:
       - DOCKER_HOST=tcp://127.0.0.1:2375
     commands:
-      - docker run -e NODE_ENV=ci -e DISABLE_CSP=true -d --net host rotm-${DRONE_COMMIT_SHA}
-      - docker run --rm --net host acceptance-${DRONE_COMMIT_SHA}
+      - docker run -e NODE_ENV=ci -e DISABLE_CSP=true -d -e EMAIL_TRANSPORT=stub -e CASEWORKER_EMAIL=test@example.com --net host rotm-${DRONE_COMMIT_SHA}
+      - docker run -e CASEWORKER_EMAIL=test@example.com --rm --net host acceptance-${DRONE_COMMIT_SHA}
```

Comment:
It _shouldn't_ be necessary to set an email address on the test runner.

Reply:
it was me taking it from r2r.  Why do we have it there?
https://github.com/UKHomeOffice/right-to-rent-check/blob/master/.drone.yml#L56

Reply:
That is a good question. I can't see a good reason for it there either. I might pull it out and see what happens.

Reply:
Cancel that. There is a "good" reason for it there, which is that some of the features `require` the steps list, which in turn `require`s the emailer behaviour, which will then error if it doesn't have an email address set.

(this isn't actually a good reason, it's a bad reason, but it's a reason nevertheless)

Reply:
We don't need it here though because the tests are better decoupled from the source code.

As an aside, try to avoid just copying and pasting code from other projects without understanding it, and how it meets the requirements you have. We lost days of work to keycloak issues last week as exactly a result of that (copying code from firearms that wasn't relevant in other contexts).